ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -195,8 +196,7 @@ namespace
         const auto& out_shape = outputs[0]->get_shape();
         const auto& in_shape = inputs[0]->get_shape();
         const auto& filter_shape = inputs[1]->get_shape();
-        Strides in_dilation(std::vector<size_t>(in_shape.size() - 2));
-        std::fill(in_dilation.begin(), in_dilation.end(), 1);
+        Strides unused; // can't be removed currently due to kmb-plugin dependency
```

Comment:
May be add a reference to a ticket in the comment?

Reply:
Done [here](https://github.com/openvinotoolkit/openvino/pull/3922/commits/fb9b1da2ed0bff53044df2a0d150791da1339fb1). I also added new overload to be able to easily switch to the new one in kmb-plugin (when this PR is merged) and later remove legacy overload when it's no longer used.